qualys-web-application-scanner/playbooks/playbooks.json

```diff
@@ -59,22 +59,19 @@
               "stepType": "/api/3/workflow_step_types/f414d039-bb0d-4e59-9c39-a8f1e880b18a"
             },
             {
-              "uuid": "0932ea6f-8c57-4307-b87b-687912e2f40c",
+              "uuid": "f1f88c30-c0c4-4245-804f-cf946a83d1e6",
               "@type": "WorkflowStep",
-              "name": "Scan Count",
+              "name": "Get Scan Count",
               "description": null,
               "status": null,
               "arguments": {
-                "name": "Qualys Web Scan",
+                "name": "Qualys Web Application Scanning(WAS)",
                 "config": "''",
```

Comment:
Change it to "config": "''" to "config": ""

Reply:
done